birdhouse/README.rst

```diff
@@ -23,22 +19,18 @@ Requirements:
   Install latest docker-ce and docker-compose for the chosen distro (not the
   version from the distro).
 
-To run ``docker-compose`` for PAVICS, the `\ ``pavics-compose.sh`` <pavics-compose.sh>`_ wrapper script must be used.
-This script will source the ``env.local`` file, apply the appropriate variable substitutions on all the configuration files ".template", and run ``docker-compose`` with all the command line arguments given to ``pavics-compose.sh``. See `\ ``env.local.example`` <env.local.example>`_ for more details on what can go into the ``env.local`` file.
+To run ``docker-compose`` for PAVICS, the `pavics-compose.sh <birdhouse/pavics-compose.sh>`_ wrapper script must be used.
```

Comment:
This is odd, browsing on github, the link is also broken as well with the addition of `birdhouse` in front of `pavics-compose.sh`.  I imagine this is to fix the RtD build?

Before you spend more time on this, is it realistic to have the same kind of relative link that would work both on github and in RtD?  Maybe we want to test this feature first.

Note on github, relative links are relative to the current file, not to the root of the checkout.

Reply:
Would it make more sense to provide the GitHub links instead?

Reply:
> Would it make more sense to provide the GitHub links instead?

I think probably should get relative links to work on GH first, then find a way to make it work on RtD.

It was working before on GH so let's not break that.  Then RtD is the added bonus.

I think we might have to copy all the files not `.rst` we reference to on RtD as well, else for sure the links won't work on RtD.  Either that or RtD need a automatic relative link rewrite to make it absolute link pointing to GH.

So basically in the code it's relative links so it works on GH browsing.  But during the doc build on RtD, those relative links are transformed to absolute link so no files need to be copied to RtD.

Hope such a feature exists.

Reply:
I think it does. It's been a LONG time since I've done some really in-depth documentation structuring. I'm gradually getting there.